CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contribution Guidelines
+Hello, and thank you for your interest in contributing to OpenScope! Please take a minute to review our Contribution Guidelines, as this will result in getting your issue resolved or pull request merged faster.
+## Getting set up
+OpenScope uses multiple tools for development, the basics being Git and Node. Getting these set up are covered in the quick start guide in README.md. To contribute, you will also need an IDE and eslint. After this, you are ready to create a branch and get developing!
+## Submitting a pull request
+Every pull request on OpenScope has a corresponding issue. Before opening a pull request, please create an issue on GitHub (if one does not already exist).
+
+For the sake of simplicity, OpenScope discourages the use of repos outside of the main OpenScope/OpenScope repository, including forks. For access to the repository, you are asked to join OpenScope's [Slack channel](https://slack.openscope.co), and contact Nate Geslin or Erik Quinn for access. After this, you can push your branch to the main repository.
+
+*Note that some members do maintain their own forks, mainly for testing purposes. There is nothing stopping you from doing so, however, PRs must be opened from a branch in the main repository.*
+
+Whenever you are ready to open the pull request itself, go to GitHub and open a new PR. There is a template in place for ease of use. While it is not required, if you decide to forgo the template, please include what issue your PR addresses ("Resolves #XXX") and a brief description as well.
+
+After your PR is opened, other contributors may approve your pull request, or request changes. After any needed changes are approved, n8rzz or erikquinn can merge your branch into the main `develop` branch.
```

Comment:
lets add a little bit about _who_ those pull requests should be assigned to:
- airports: @Fechulo 
- everything else: @erikquinn and/or @n8rzz

Reply:
@n8rzz Airports should be assigned to the airport team shouldn't they?

Reply:
@indianbhaji that is correct

Reply:
done